eslint.config.mjs

```diff
@@ -0,0 +1,140 @@
+import typescriptEslint from "@typescript-eslint/eslint-plugin";
```

Comment:
In het voorbeeld https://typescript-eslint.io/getting-started/#step-2-configuration staat op de eerste regel een `// @ts-check` directive. Dan wordt deze JavaScript file ook door TypeScript geanalyseerd.

Reply:
Dit heb ik nog gedaan